src/main/java/Duke.java

```diff
@@ -69,7 +71,7 @@ public static void main(String[] args) throws IOException {
                 }
                 printLines();
             }
-        }
+        } while(response.compareTo("bye") != 0);
```

Comment:
just nitpicking, reserved words (e.g. "while") should be followed with white spaces